lib/api/src/store.rs

```diff
@@ -41,6 +41,19 @@ impl Store {
         *m = handler;
     }
 
+    /// Catches any Wasm traps that happen within the execution of `closure`,
+    /// returning them as a `Result`.
+    ///
+    /// # Safety
+    ///
+    /// Highly unsafe since `closure` won't have any dtors run.
```

Comment:
```suggestion
    /// Highly unsafe since `closure` won't have any destructors run.
```

Reply:
Please, can you add an example of `catch_traps` + `Function::call_unsafe` here? That's the perfect place to provide an example (if possible, a doctest).